### PR TITLE
feat(timeline): new timeline component

### PR DESCRIPTION
**Pull request by DanSallau**

### CODE REVIEW

File: lib/src/timeline/timeline.component.scss [hunk 1]
```diff
@@ -0,0 +1,338 @@
+$white: #fff;
+$gray-100: #f8f8f8;
+$gray-300: #dedede;
+$blue: #41b0ee;
+$blue-darker: #007ac7;
```
Comment (yousifalraheem) [inline lib/src/timeline/timeline.component.scss:5]:
```suggestion
@import "~@sebgroup/bootstrap/scss/core";
```

**Comment by mario-subo** [comment]
Have you made any changes from Angular Components that need my attention? Did you copy over the unit tests and are they passing?

**Pull request merged**